src/entity/CommonGame.java

```diff
@@ -0,0 +1,108 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class CommonGame implements Game {
+    private final String ID;
+    private int score;
+    private final String difficulty;
+    private final String genre;
+    private final int initialLives;
+    private final int maxRounds;
+    private int currentLives;
+    private int roundsPlayed = 0;
+    private Round round;
+    private ArrayList<Round> allRounds;
+
+    private final LocalDateTime createdAt;
+    private LocalDateTime FinishedAt;
```

Comment:
```suggestion
    private LocalDateTime finishedAt;
```
Same thing in `getFinishedAt()` and `setFinishedAt()`.